### DISCUSSION TITLE
Let op: Engelse termen bij gebruik API

### DISCUSSION THREAD

**Discussion by CelineJansen** [kadaster-labs/sensrnet-home#265] [Show and tell]
Bij gebruik van de API (https://demo.sensorenregister.nl/api/#/Datastream/DatastreamController_addDatastream) is het van belang om thema's in het Engels op te voeren. "Geluid" werkt niet, "Sound" wel.

**Comment by marcvanandel** [comment]
Klopt! Zie ook https://github.com/kadaster-labs/sensrnet-registry-frontend/issues/83 ... en is inderdaad te weinig goed gedocumenteerd, zeker voor wat betreft waardelijsten 😬
